### ISSUE TITLE
🐛 [BUG]: Upon using Bard-ai for a few minutes or so, it will invalidate the cookie or return NULL

### ISSUE THREAD

**Issue by malice-cli** [EvanZhouDev/bard-ai#61]
### Prerequisites

- [X] I have searched the [issues](https://github.com/EvanZhouDev/bard-ai/issues) and believe that it has not already been reported
- [X] It occurs on the latest version of `bard-ai`
- [X] I have read the [FAQ](https://bard-ai.js.org/faq/) and there are no solutions there

### Bug Description

As said in the title, when you are using it for more than 5 minutes, it will start returning null responses or have a invalidated cookie.
![image](https://github.com/EvanZhouDev/bard-ai/assets/95049678/c8df7d27-8c2c-4748-b817-6ddfad51ff89)
and I use a new cookie every time, it would work, but after 5 minutes, it gets invalidated.
Im not sure if this is a issue with the library itself, or bard.

### Reproduction Steps

1. Getting a new cookie
2. Using bard for 5 minutes or so
3. It gets invalidated and returns null

### Occurance Rate

I can reproduce this bug 100% of the time.

**Comment by EvanZhouDev** [comment]
Are you using the __Secure-1PSIDTS cookie as well? See docs here: https://bard-ai.js.org/faq/#cant-use-bard

**Comment by malice-cli** [comment]
yes. it still returns a invalidation.

**Comment by ziuss76** [comment]
I have same error here, using __Secure-3PSID and __Secure-3PSIDTS, and few minutes after it returns same error above.

**Comment by jonvicbarcenas** [comment]
Is there a way to auto fetch cookies using gmail and passwords? It invalidates mine after minutes

**Comment by Goldp-js** [comment]
After some research I saw that in this line where all the login is in charge, it looks for variable 1, and then when the cookie does not work it will return Error: TypeError: Cannot read properties of null (reading '1') 
 
```javascript
const SNlM0e = ResponseText.match(/SNlM0e":"(.*?)"/)[1];
```

**Comment by Goldp-js** [comment]
Possibly it is a captcha interfering or something, since the interface bard asks me to solve several captchas when I use cookies.

**Comment by Goldp-js** [comment]
The problem is that the cookies mentioned change constantly and since it is not a good idea to automate that, since, if the process of obtaining cookies could be automated, a captcha will always appear on the bard website so it is not possible to work in obtaining automatic cookies.

**Comment by EvanZhouDev** [comment]
Yes, please migrate to the PaLM API. The simulated Bard API interface is no longer stable (it's only working for some people)

**Comment by jonvicbarcenas** [comment]
I use Node-fetch and after a few hours, the PSID key starts working again. I use the Kiwi browser on my phone then incognito to retrieve cookies because in Chrome on PC using incognito mode, they expire. If you saw PSIDTS on your cookie editor, your psid will not work even after the limit cooldown

**Comment by kevindevm** [comment]
yes please is very bad as it is
